### ISSUE TITLE
Login failed due to 401

### ISSUE THREAD

**Issue by tianchengbaihe** [uwolfer/gerrit-intellij-plugin#255]
We use gerrit 2.6.1.
And I input the correct url, username, password.
I got error message when I click Test button:

`
<title>Error 401 Unauthorized</title>
</head>
<body>
<h2>HTTP ERROR: 401</h2>
<p>Problem accessing /a/accounts/self. Reason:
<pre>    Unauthorized</pre></p>
<hr /><i><small>Powered by Jetty://</small></i>
`

**Comment by uwolfer** [comment]
Have you also tried to set a password (HTTP) in Gerrit settings and then configure this one in IntelliJ Gerrit plugin settings?

**Comment by tianchengbaihe** [comment]
Yes, I copied the password from setting page of gerrit in browser.
I haven't tried it later, and today I tried again, and got(when I click test button):
```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>401 Authorization Required</title>
</head><body>
<h1>Authorization Required</h1>
<p>This server could not verify that you
are authorized to access the document
requested.  Either you supplied the wrong
credentials (e.g., bad password), or your
browser doesn't understand how to supply
the credentials required.</p>
</body></html>.
```
Since months past, and I still can't find similar issues raised here, it seems really a strange case, maybe the server requires some specified setting?

Thanks

**Comment by uwolfer** [comment]
How is the login handled for the web UI? Is it some type of single-sign-on solution or HTTP-access? Have you also tried to use the same credentials as you use for web UI login?

**Comment by tianchengbaihe** [comment]
Thanks. I got the cause!
I have a password when the account created. suppose the password is A.
I changed the password to B after I first time login gerrit in web UI.
Everytime I login through web UI, I use the password B.
And When I use gerrit plugin, I also input the password B which cause the errors in our discussion.
Today I used the password A in gerrit plugin and the login successful.
It means after I changed the password, I need to use the newest password in web ui, and the original password in gerrit plugin.
It seems like we configure the gerrit wrongly.

Hope this comment would help others if encounter similar situation.
